algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -172,13 +180,16 @@ class CMSContraMapSpec extends WordSpec with Matchers with GeneratorDrivenProper
     val a = 4
     val b = 0
     val width = 1234
-    val x = Array(1.toByte).toSeq // same as Seq(1.toByte)
-    targetHasher.hash(a, b, width)(x) should be(677)
+    val x = Array(113.toByte).toSeq // same as Seq(1.toByte)
```

Comment:
This changed with the new hashing code -- the previous test had hardcoded the old hash result, so it needed to change anyway since we are using a different hasher.